neuromation/cli/formatters/jobs.py

```diff
@@ -96,6 +96,19 @@ def __call__(self, job_status: JobDescription) -> str:
         resource_formatter = ResourcesFormatter()
         result += resource_formatter(job_status.container.resources) + "\n"
         result += f"Preemptible: {job_status.is_preemptible}\n"
+
+        if job_status.container.volumes:
+            rows = [
+                (
+                    volume.container_path,
+                    f"{volume.storage_uri}",
+                    "RO" if volume.read_only else " ",
```

Comment:
Could you please expand *RO* into *READONLY* or something like this?

Looks like we have enough space on the screen for a little longer but much better understandable line.